tests/test_cmdline.py

```diff
@@ -0,0 +1,31 @@
+import zipline.__main__ as main
+from zipline.testing import ZiplineTestCase
+from zipline.testing.predicates import (
+    assert_equal,
+    assert_raises_str,
+)
+
+
+class CmdLineTestCase(ZiplineTestCase):
```

Comment:
we should add some tests that actually invoke to the entry point to make sure we parse the args correctly when running. Click has some helpers for testing applications which we can take advantage of: http://click.pocoo.org/5/testing/